docs/README-EDITING.md

```diff
@@ -0,0 +1,174 @@
+
+```js
+// Wrong:
+/** {@link Options#autoColumnSize}. */
+
+// OK: 
+/** {@link options#autocolumnsize Options#autoColumnSize}. */
+
+
+// Wrong:
+/** {@link HiddenRows} */
+
+// OK:
+/** {@link hidden-rows HiddenRows} */
+```
+
+## Documentation versioning
+
+To create a new version of the Handsontable documentation:
+
+* From the `handsontable/docs` directory, run:
+```bash
+npm run docs:version <semver.version>
+# for example:
+# npm run docs:version 9.0
+```
+
+To remove an existing version of the Handsontable documentation:
+
+* Remove the required version's [directory](./README.md#handsontable-docs-directory-structure):
+```bash
+rm -rf ./<semver.version>
+```
+
+## Markdown containers
+
+To render content in different ways, the documentation uses custom Markdown containers, for example:
+
+```markdown
+::: example #exampleId .class :preset --html 1 --js 2
+
+// code example
+
+:::
+```
+
+We use the following Markdown containers:
+
+| Container                    | Usage                                                      |
+|------------------------------|------------------------------------------------------------|
+| `::: tip`                    | A note.                                                    |
+| `::: example [options]`      | Renders a code example as specified in [options].          |
+| `::: source-code-link [URL]` | Adds a source code link to the right of an API ref header. |
+
+### Adding code examples
+Using the `example` Markdown container, you can add code snippets that show the code's result:
+
+```js
+::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
+// `#example1` is the example's ID, for creating a Handsontable container
+    ```html
+    <div id="example1"></div>
+    ```
+    ```js
+    // code here
+    ```
+    ```css
+    /* custom css here */
+    ```
+:::
+```
+
+The `example` Markdown container offers the following options:
+
+| Option         | Required | Example         | Possible values                                            | Usage                                                                                              |
+|----------------|----------|-----------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `#exampleId`   | Yes      | `#data-grid-1`  | String                                                     | Container's unique ID.                                                                             |
+| `.class`       | No       | `.new-class`    | String                                                     | Container's custom class.                                                                          |
+| `:preset`      | No       | `:react`        | [`preset`](.vuepress/handsontable-manager/dependencies.js) | Sets code dependencies.                                                                            |
```

Comment:
If I use an incorrect preset, I got the following error:
```
./next/guides/columns/column-header.md
Module build failed (from ./node_modules/@vuepress/markdown-loader/index.js):
TypeError: Cannot read property 'map' of undefined
    at getDependencies (C:\Users\Piotr\projects\handsontable\docs\.vuepress\handsontable-manager\dependencies.js:98:28)
```
What helps me nothing to understand where I made a mistake.